test-forge.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$ETH_RPC_URL" && "$(seth chain)" == "ethlive" ]] || { echo "Please set a mainnet ETH_RPC_URL"; exit 1; }
+
+for ARGUMENT in "$@"
+do
+    KEY=$(echo $ARGUMENT | cut -f1 -d=)
+    VALUE=$(echo $ARGUMENT | cut -f2 -d=)
+
+    case "$KEY" in
+            match)  MATCH="$VALUE" ;;
+            block)  BLOCK="$VALUE" ;;     
+            *)   
+    esac
+done
+
+if [[ -z "$MATCH" && -z "$BLOCK" ]]; then
+    forge test --fork-url "$ETH_RPC_URL" --force
+elif [[ -z "$BLOCK" ]]; then
+    forge test --fork-url "$ETH_RPC_URL" --match "$MATCH" -vvv --force
+elif [[ -z "$MATCH" ]]; then
+    forge test --fork-url "$ETH_RPC_URL" --fork-block-number "$BLOCK" --force
```

Comment:
nit: might be worth adding a bit of verbosity by default, to get some logs, for quick debugging.
```suggestion
    forge test --fork-url "$ETH_RPC_URL" --vv --force
elif [[ -z "$BLOCK" ]]; then
    forge test --fork-url "$ETH_RPC_URL" --match "$MATCH" -vvv --force
elif [[ -z "$MATCH" ]]; then
    forge test --fork-url "$ETH_RPC_URL" --fork-block-number "$BLOCK" --vv --force
```